README.md

```diff
@@ -1,41 +1,82 @@
+  <a href="#installation">Installation</a> ◈
+  <a href="#getting-started">Getting started</a> ◈
+  <a href="#supporting-the-project">Supporting the project</a> ◈
+  <a href="#joining-the-discussion">Joining the discussion</a> 
+</p>
+
+---
+
+## About
+
+This is the **official** IOTA wallet, which allows you to do the following:
+* Create password-protected accounts to store and access your seeds
+* Send transactions
+* Generate addresses
+* Read your balance and transaction history
+
+This is beta software, so there may be performance and stability issues.
+Please report any issues in our [issue tracker](https://github.com/iotaledger/trinity/issues/new).
 
-[![Dependabot Status](https://api.dependabot.com/badges/status?host=github&repo=iotaledger/trinity-wallet)](https://dependabot.com)
+## Prerequisites
 
+To compile Trinity, you must have the following:
 
-This is the shared repository for the IOTA Trinity Mobile and Desktop wallets. The apps are based on React and built with React-Native (mobile) and Electron (desktop).
+* Node.js 8 or higher. We recommend the [latest LTS](https://nodejs.org/en/download/)
+* [Yarn](https://yarnpkg.com/) package manager.
 
-## Prerequisites
+## Installation
 
-- Node.js (8+)
-- Yarn
+To install the dependencies needed to compile both Trinity Mobile and Trinity Desktop, do the following:
 
-## Instructions
+1. Clone this repository
 
-1. Clone this repo
-```
-git clone https://github.com/iotaledger/trinity-wallet.git
-```
+    ```
+    git clone https://github.com/iotaledger/trinity-wallet.git
+    ```
 
-2. Go to the trinity-wallet directory
-```
-cd trinity-wallet
-```
+2. Change into the `trinity-wallet` directory
+
+    ```
+    cd trinity-wallet
+    ```
 
 3. Install the shared dependencies
-```
-yarn && yarn deps:shared
-```
 
-4. Follow the instructions to install dependencies for [Trinity Mobile](https://github.com/iotaledger/trinity-wallet/blob/develop/src/mobile/README.md) or [Trinity Desktop](https://github.com/iotaledger/trinity-wallet/blob/develop/src/desktop/README.md)
+    ```
+    yarn && yarn deps:shared
+    ```
+
+## Getting started
 
+To compile Trinity, follow the instructions for either [Trinity Mobile](https://github.com/iotaledger/trinity-wallet/blob/develop/src/mobile/README.md) or [Trinity Desktop](https://github.com/iotaledger/trinity-wallet/blob/develop/src/desktop/README.md).
 
-## Branches
+## Supporting the project
 
-To create a new feature or bugfix (or chore) please create a new branch and use a prefix (e.g `feature/my-awesome-new-feature` or `bugfix/something-not-working`). The prefixes we use are `bug`, `feat`, `chore` and `hotfix`. Please don't use anything else.
+If Trinity has been useful to you and you feel like contributing, consider posting a [bug report](https://github.com/iotaledger/trinity-wallet/issues/new), [feature request](https://github.com/iotaledger/trinity-wallet/issues/new) or a [pull request](https://github.com/iotaledger/trinity-wallet/pulls/).
```

Comment:
```suggestion
If Trinity has been useful to you and you feel like contributing, consider posting a [bug report](https://github.com/iotaledger/trinity-wallet/issues/new?labels=T+-+Bug&template=bug_report.md&title=), [feature request](https://github.com/iotaledger/trinity-wallet/issues/new?labels=&template=feature_request.md&title=) or a [pull request](https://github.com/iotaledger/trinity-wallet/pulls/).
```
Updated links so that they direct readers to the appropriate templates